docs/partials/index.md

```diff
@@ -106,6 +106,22 @@ This common example uses the default `auro-dropdown` element with the attributes
 
 </auro-accordion>
 
+#### Truncated trigger component with fixed width
```

Comment:
Let's move this to docs/partials/api.md and put it with the "Other Examples" section of the page.

It does not belong here as a subjection of this heading.

Reply:
ok @jason-capsule42, thanks.